packages/frontend/src/components/common/Table/TableProvider.tsx

```diff
@@ -67,13 +89,53 @@ export const TableProvider: FC<Props> = ({ children, ...rest }) => {
         setTempColumnOrder([ROW_NUMBER_COLUMN_ID, ...(columnOrder || [])]);
     }, [columnOrder]);
 
+    //TODO fix weird borderless cell
+    //TODO pivots ?
+    const rowColumnWidth = (rowColumn.meta?.width || 30) + 10;
+    const frozenColumns = columns.filter((col) => col.meta?.frozen);
+    const frozenColumnWidth = 90; // TODO this should be dynamic
```

Comment:
it does not work without specifying the fixed width?

Reply:
we need to give a `left` if we don't want sticky columns to overlap each other, the easiest way to ensure the left is correct ist o give a width, but at this point in the code the size of the columns are not calculated, so all I can do is to give it a fixed size